backend/package.json

```diff
@@ -26,10 +26,13 @@
     "@nestjs/mongoose": "^7.2.4",
     "@nestjs/platform-express": "^7.6.13",
     "bcrypt": "^5.0.1",
+    "ioredis": "^4.23.0",
```

Comment:
This is not used anywhere in the code-base, it should be safe to uninstall it right?